frontend/src/app/shared/components/sidebar/sidebar.component.ts

```diff
@@ -1,8 +1,50 @@
+import { HttpEventType, HttpResponse } from '@angular/common/http';
 import { Component } from '@angular/core';
+import { HttpInternalService } from '@core/services/http-internal.service';
+import { NotificationService } from '@core/services/notification.service';
 
 @Component({
     selector: 'app-sidebar',
     templateUrl: './sidebar.component.html',
     styleUrls: ['./sidebar.component.sass'],
 })
-export class SidebarComponent { }
+export class SidebarComponent {
+    private readonly staticFilesRoutePrefix = '/api/staticfiles';
```

Comment:
Maybe it would be better to create a separate service for download files instead of add this logic to SidebarComponent?